profiling/src/profile/mod.rs

```diff
@@ -1020,6 +1112,45 @@ mod api_test {
         assert_eq!(label.num, 101);
         assert_eq!(str, "");
         assert_eq!(num_unit, "");
+
+        // For some reason, this test never covered index 1
```

Comment:
Minor: This is a slightly weird comment to me... E.g. I'm not sure it's interesting going forward for this fact to be documented in the code, since you've fixed it now :)

Reply:
Fixed